From 7bba9ea90ad3548a9d3fd5796f2390b545e2fdee Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 04 Mar 2021 20:49:50 +0200
Subject: [PATCH] [SW-31170] greco: replace goya2 to greco

As goya2 is renamed to greco, this patch add changes to add
conditions in makefile and headers required for greco.

Change-Id: I0c9cc6c1b64a25c07668a6b9ba92d5405f060949
---

diff --git a/Makefile b/Makefile
index 732b5fd..37a4a52 100644
--- a/Makefile
+++ b/Makefile
@@ -54,10 +54,10 @@
 PCI_PHY_FW_TARGET = $(BINDIR)/pcie_phy_fw.bin
 
 # compile binaries based on target
-ifeq ($(HL_DEV),goya2_arc)
+ifeq ($(HL_DEV),$(filter $(HL_DEV),goya2_arc greco_arc))
 	BINARIES = $(PPBOOT_TARGET) $(PCI_PHY_FW_TARGET)
 	LDFLAGS += -Wl,--just-symbols ../arcboot-custom/exports.elf
-else ifeq ($(HL_DEV),goya2_arc_gnu)
+else ifeq ($(HL_DEV),$(filter $(HL_DEV),goya2_arc_gnu greco_arc_gnu))
 	BINARIES = $(PREBOOT_TARGET)
 else
 	BINARIES = $(PPBOOT_TARGET) $(PREBOOT_TARGET) $(PCI_PHY_FW_TARGET)
diff --git a/preboot/thermal_v2.c b/preboot/thermal_v2.c
index 8f70206..a24228e 100644
--- a/preboot/thermal_v2.c
+++ b/preboot/thermal_v2.c
@@ -5,7 +5,7 @@
  *
  */
 
-#include <goya2/goya2_reg_map.h>
+#include <greco/greco_reg_map.h>
 #include <goya2_blocks_linux_driver.h>
 #include <psoc_global_conf_regs.h>
 #include <dcon0_tstd_masks.h>
